common/test/java/com/couchbase/lite/DocumentTest.java

```diff
@@ -1928,6 +1929,41 @@ public void testGetExpirationOnDocInACollectionOfClosedDatabase() throws Couchba
         }
     }
 
+    // Test setting expiration on doc in a collection of deleted database throws CBLException
+    @Test
+    public void testSetExpirationOnDocInCollectionOfDeletedDB() {
+        Date expiration = new Date(System.currentTimeMillis() + 30000L);
+        try {
+            String id = "doc_id";
+            MutableDocument document = new MutableDocument(id);
+            deleteDb(baseTestDb);
```

Comment:
This test can fail in *VERY* misleading ways.  The `deleteDb` could throw a CBLException that is not the right kind.  that would cause the test to fail, but for the entirely wrong reason.  Move everything that isn't being tested outside the try block.  You'll have to add CBLException to the signature.